views/incorrect-information/wrong-statement-of-capital.html

```diff
@@ -26,14 +26,12 @@ <h1 class="govuk-heading-xl">
             <p>The company's share capital does not match the shares held by its shareholders.</p>
           {% endif %}
 
-          <p>You must update this information by filing a confirmation statement through
-            <a target="_blank" rel="noopener noreferrer" href="https://ewf.companieshouse.gov.uk//seclogin?tc=1" data-event-id="webfiling-service-link-from-invalid-share-or-amount-unpaid-totals-wrong-statement-of-capital-page">our WebFiling service</a>. You will need to sign in to or create a WebFiling account to do this.</p>
-
-        {% else %}
-          <p>Currently, changes to share capital can only be made by filing a confirmation statement through
-            <a target="_blank" rel="noopener noreferrer" href="https://ewf.companieshouse.gov.uk//seclogin?tc=1" data-event-id="webfiling-service-link-from-wrong-statement-of-capital-page">our WebFiling service (opens in a new tab)</a>. You will need to sign in to or create a WebFiling account to do this.
-          </p>
         {% endif %}
+
```

Comment:
line 19 & 29
{% if totalAmountUnpaidValidation === false or sharesValidation === false %}
is not needed anymore as it's contents each have their own if.

Reply:
Are you sure? It will clash with you comment above about the matomo events.

Reply:
just noticed it in the PR as it stands now.
currently it is 
if (a or b) {
 if (a) {..}
 if (b) {..}
}

so you can just do 
if(a) {..}
if (b) {..}

might want to keep it if you are adding the else block back, depends how you want to build the url